dace/transformation/dataflow/redundant_array.py

```diff
@@ -325,7 +325,7 @@ def can_be_applied(graph, candidate, expr_index, sdfg, strict=False):
         else:
             # Two views connected to each other
             if isinstance(in_desc, data.View):
-                return False
+                return True
```

Comment:
Isn't this case handled by some new transformation that I saw recently (squeeze/unsqueeze view remove)? If not, we still should make it a separate transformation. RedundantArray is overloaded and it will be very difficult in the future to maintain.